var/spack/repos/builtin/packages/py-htgettoken/package.py

```diff
@@ -24,8 +24,10 @@ class PyHtgettoken(PythonPackage):
 
     # The following versions refer to setuptools-buildable commits after 1.16;
     # they are special reproducible version numbers from `git describe`
+    version("2.0-2", commit="e777edb54eb28194351aee0a818aa70489017ab0")
```

Comment:
The comment refers to the 1.16 versions only.
```suggestion
    version("2.0-2", commit="e777edb54eb28194351aee0a818aa70489017ab0")
    # The following versions refer to setuptools-buildable commits after 1.16;
    # they are special reproducible version numbers from `git describe`
```